mm2src/mm2_main/src/rpc.rs

```diff
@@ -28,14 +28,11 @@ use futures::future::{join_all, FutureExt};
 use http::header::{HeaderValue, ACCESS_CONTROL_ALLOW_ORIGIN, CONTENT_TYPE};
 use http::request::Parts;
 use http::{Method, Request, Response, StatusCode};
-use lazy_static::lazy_static;
 use mm2_core::mm_ctx::MmArc;
 use mm2_err_handle::prelude::*;
 use mm2_rpc::mm_protocol::{MmRpcBuilder, MmRpcResponse, MmRpcVersion};
-use regex::Regex;
```

Comment:
we dropped the only usage of regex here,
we can also drop it in: https://github.com/KomodoPlatform/komodo-defi-framework/blob/079ea5e4fd9929a73a6f3a7f62240933d7537a43/mm2src/mm2_main/Cargo.toml#L80-L82